### DISCUSSION TITLE
How to Select All ?

### DISCUSSION THREAD

**Discussion by pranavrbm** [microsoft/vscode-jupyter#15375] [Questions and Answers] [answered]
when I went through many resources it says 
"1. Click on the first cell you want to copy.
2. Scroll down to the last cell you want to copy.
3. Hold down the Shift key on your keyboard.
4. While holding the Shift key, click on the last cell you want to copy."
but I am unable to do it 
What should I do?

**Comment by DonJayamanne** [comment, marked as answer]
Press Ctrl+ A to select all cells
Ensure you have clicked on a cell (outside a cell editor)

**Reply by pranavrbm** [reply]
Thank you for answering but it is still not working 
nothing works except shift+(down or up arrow  button)
